test-data-server/src/services/single-data.service.ts

```diff
@@ -265,22 +265,20 @@ export class SingleData implements IDatabase {
     }
 
     async getScheduledPaymentsForAccount(customerId: string, accountId: string, query: any): Promise<BankingScheduledPaymentV2[]> {
-        let ret: any = {};
+        //let ret: any = {};
```

Comment:
if retArray is now returned instead of 'ret', might as well remove this commented code defining 'ret'.